TemporalSystem.cpp

```diff
@@ -1211,6 +1211,28 @@ HOOK_METHOD(CloneSystem, OnLoop, () -> void)
 {
     LOG_HOOK("HOOK_METHOD -> CloneSystem::OnLoop -> Begin (TemporalSystem.cpp)\n")
     g_dilationAmount = GetRoomDilationAmount(g_sysDilationRooms, roomId);
+
+    float increment = (G_->GetCFPS()->GetSpeedFactor() * 0.0625);
+    float cloneSpeedMultiplier = 1;
+    auto crewList = G_->GetShipManager(_shipObj.iShipId)->vCrewList;
+    G_->GetCrewFactory()->GetCloneReadyList(crewList,(_shipObj.iShipId==0));
+    int deadCrewCount=G_->GetCrewFactory()->CountCloneReadyCrew(_shipObj.iShipId==0);
```

Comment:
This is redundant when we're already calling GetCloneReadyList; we can just call size() on the clone list.